protocols/migration/participants.py

```diff
@@ -7,6 +7,60 @@
 from protocols.migration import BaseMigration
 
 
+class MigrationParticipants100To104SNAPSHOT(BaseMigration):
+    old_model = participant_1_0_0
+    new_model = participant_1_0_4_SNAPSHOT
+
+    def migrate_cancer_participant(self, cancer_participant):
+        migrated_participant = self.new_model.CancerParticipant.fromJsonDict(cancer_participant.toJsonDict())
+
+        migrated_participant.yearOfBirth = 0
+
+        if cancer_participant.primaryDiagnosisDisease is not None:
+            migrated_participant.primaryDiagnosisDisease = [cancer_participant.primaryDiagnosisDisease]
+
+        if cancer_participant.primaryDiagnosisSubDisease is not None:
+            migrated_participant.primaryDiagnosisSubDisease = [cancer_participant.primaryDiagnosisSubDisease]
+
+        if cancer_participant.assignedICD10 is not None:
+            migrated_participant.assignedICD10 = [cancer_participant.assignedICD10]
+
+        migrated_participant.tumourSamples = self.migrate_tumour_samples(cancer_participant.tumourSamples)
+        migrated_participant.germlineSamples = self.migrate_germline_samples(cancer_participant.germlineSamples)
+
+        return self.validate_object(
+            object_to_validate=migrated_participant, object_type=self.new_model.CancerParticipant
+        )
+
+    def migrate_tumour_sample(self, tumour_sample):
```

Comment:
LDPCode in `CancerParticipant-1.0.0` may be fit within LDPCode under `TumourSample-1.0.4-SNAPSHOT` ??? @Antonior26 ?

Reply:
That's a good point @priesgo 
The LDPCode in participant 1.0.0 is now in GermlineSample and TumourSample in 1.0.4
Previously @Antonior26 said to take the LDPCode from GermlineSample so I'll do that here.
I'll add an explicit test